File: MS/src/main/java/com/yskc/ms/entity/ms/ServiceLog.java
package com.yskc.ms.entity.ms;

import com.baomidou.mybatisplus.annotations.TableId;
import com.baomidou.mybatisplus.annotations.TableName;

import java.math.BigDecimal;
import java.util.Date;

/**
 * Created by hck
 * on 2020/5/5 13:52
 * 客户服务表
 */
@TableName("serviceLog")
public class ServiceLog {
    private static final long serialVersionUID = 1L;

    @TableId
    private Integer id;
    private Integer creatorId;
    private Integer lastUpdatorId;
    private Date createTime;
    private Date lastUpdateTime;

    private Integer customerId;//客户id
    private Integer serviceType;//服务类型
    private BigDecimal totalTime;//服务总工时
    private Integer totalStaff;//服务总人数
    private String lastProblem;//之前遇到问题及解决方案
    private String nowProblem;//本次遇到的问题
    private String customerAdvice;//客户建议
    private String customerParticipan;//客户参与人员
    private String serviceDept;//服务的具体车间/部门
    private Integer status;//提交状态0未提交1提交
    private Date startDate;//服务开始时间
    private Date endDate;//服务结束时间
    private String auditOpinion;
    private String filePath;
    private Date submitDate;
    private Integer days;
    public void setId(Integer id) {
        this.id = id;
    }

    public String getAuditOpinion() {
        return auditOpinion;
    }

    public void setAuditOpinion(String auditOpinion) {
        this.auditOpinion = auditOpinion;
    }

    public String getFilePath() {
        return filePath;
    }

    public void setFilePath(String filePath) {
        this.filePath = filePath;
    }

    public Date getStartDate() {
        return startDate;
    }

    public void setStartDate(Date startDate) {
        this.startDate = startDate;
    }

    public Date getEndDate() {
        return endDate;
    }

    public void setEndDate(Date endDate) {
        this.endDate = endDate;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public Integer getCreatorId() {
        return creatorId;
    }

    public void setCreatorId(Integer creatorId) {
        this.creatorId = creatorId;
    }

    public Integer getLastUpdatorId() {
        return lastUpdatorId;
    }

    public void setLastUpdatorId(Integer lastUpdatorId) {
        this.lastUpdatorId = lastUpdatorId;
    }

    public Date getCreateTime() {
        return createTime;
    }

    public void setCreateTime(Date createTime) {
        this.createTime = createTime;
    }

    public Date getLastUpdateTime() {
        return lastUpdateTime;
    }

    public void setLastUpdateTime(Date lastUpdateTime) {
        this.lastUpdateTime = lastUpdateTime;
    }

    public Integer getCustomerId() {
        return customerId;
    }

    public void setCustomerId(Integer customerId) {
        this.customerId = customerId;
    }

    public Integer getServiceType() {
        return serviceType;
    }

    public void setServiceType(Integer serviceType) {
        this.serviceType = serviceType;
    }

    public BigDecimal getTotalTime() {
        return totalTime;
    }

    public void setTotalTime(BigDecimal totalTime) {
        this.totalTime = totalTime;
    }

    public Integer getTotalStaff() {
        return totalStaff;
    }

    public void setTotalStaff(Integer totalStaff) {
        this.totalStaff = totalStaff;
    }

    public String getLastProblem() {
        return lastProblem;
    }

    public void setLastProblem(String lastProblem) {
        this.lastProblem = lastProblem;
    }

    public String getNowProblem() {
        return nowProblem;
    }

    public void setNowProblem(String nowProblem) {
        this.nowProblem = nowProblem;
    }

    public String getCustomerAdvice() {
        return customerAdvice;
    }

    public void setCustomerAdvice(String customerAdvice) {
        this.customerAdvice = customerAdvice;
    }

    public String getCustomerParticipan() {
        return customerParticipan;
    }

    public void setCustomerParticipan(String customerParticipan) {
        this.customerParticipan = customerParticipan;
    }

    public String getServiceDept() {
        return serviceDept;
    }

    public void setServiceDept(String serviceDept) {
        this.serviceDept = serviceDept;
    }

    public Integer getStatus() {
        return status;
    }

    public void setStatus(Integer status) {
        this.status = status;
    }

    public Date getSubmitDate() {
        return submitDate;
    }

    public void setSubmitDate(Date submitDate) {
        this.submitDate = submitDate;
    }

    public Integer getDays() {
        return days;
    }

    public void setDays(Integer days) {
        this.days = days;
    }
}

File: YSDB/MSDB/Models/serviceLog.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace MSDB.Models
{
    /// <summary>
    /// 客户服务记录
    /// </summary>
    public class serviceLog : tablebase
    {
        [Key]
        public int id { get; set; }
        /// <summary>
        /// 客户id
        /// </summary>
        public int customerId { get; set; }
        /// <summary>
        /// 服务方式0现场对接1电话沟通2其他
        /// </summary>
        public int serviceType { get; set; }
        /// <summary>
        /// 服务总工时
        /// </summary>
        [Column(TypeName = "decimal(18,2)")]
        public decimal? totalTime { get; set; }
        /// <summary>
        /// 服务总人数
        /// </summary>
        public int totalStaff { get; set; }
        /// <summary>
        /// 上次问题解决情况
        /// </summary>
        [MaxLength(800)]
        public String lastProblem { get; set; }
        /// <summary>
        /// 本次遇到问题及解决方案
        /// </summary>
        [MaxLength(800)]
        public String nowProblem { get; set; }
        /// <summary>
        ///客户建议
        /// </summary>
        [MaxLength(800)]
        public String customerAdvice { get; set; }
        /// <summary>
        ///客户主要参与人员
        /// </summary>
        [MaxLength(100)]
        public String customerParticipan { get; set; }
        /// <summary>
        ///本次服务具体的车间/部门/工艺段
        /// </summary>
        [MaxLength(100)]
        public String serviceDept { get; set; }
        /// <summary>
        /// 是否提交0未提交1提交
        /// </summary>
        public int status { get; set; }
        /// <summary>
        /// 服务开始时间 
        /// </summary>
        public DateTime startDate { get; set; }
        /// <summary>
        /// 服务结束时间
        /// </summary>
        public DateTime endDate { get; set; }

        /// <summary>
        ///审核意见
        /// </summary>
        [MaxLength(500)]
        public String auditOpinion { get; set; }
        /// <summary>
        ///上传地址
        /// </summary>
        [MaxLength(500)]
        public String filePath { get; set; }
        /// <summary>
        /// 提交时间
        /// </summary>
        public DateTime? submitDate { get; set; }
        ///<summary>
        ///服务天数
        ///</summary>
        public int days { get; set; }

    }
}
